docs/UserGuide.adoc

```diff
@@ -311,6 +311,31 @@ Clears all saved links from the manager.
 
 Format: `clear`
 
+=== List links in reader: `readerlist` `[coming in v2.0]`
+
+Lists all the hyperlinks in the reader.
+
+Format: `readerlist`
+
+Examples:
+
+- `readerlist` Lists all the hyperlinks in the reader.
+
+=== Add reader links: `readeradd` `[coming in v2.0]`
```

Comment:
This command is not needed, because `readerlist` can be implemented as a shortcut command to `search {current_link}`. The `search WEBPAGE_URL` command will show all the links contained in that web page, and users can use `add` from there to add links.

Reply:
Makes sense!